gammapy/datasets/tests/test_make.py

```diff
@@ -15,3 +16,10 @@ def test_make_test_psf_fits_table():
     psf2 = psf.psf_at_energy_and_theta(energy, theta)
     fraction = psf2.containment_fraction(0.1)
     assert fraction == 0.44485638998490795
+
+
+def test_make_test_observation_table():
+    observatory_name='HESS'
+    n_obs = 10
+    obs_table = make_test_observation_table(observatory_name, n_obs)
+    assert len(obs_table) == n_obs
```

Comment:
Maybe add a few more asserts to check that positions and times are roughly as expected, i.e. that they were filled correctly?